scripts/sonar/sonar-scan.sh

```diff
@@ -13,10 +14,10 @@ SONAR_LOGIN=""
 SONAR_ORG="hootenanny"
 SONAR_PROJECT="hoot"
 SONAR_SOURCES="./hoot-core,./hoot-js,./hoot-rnd,./tbs,./tgs"
-SONAR_THREADS="$(nproc)"
+SONAR_THREADS="$(( $(nproc) * 2 ))"  # Sonar scan doesn't max out CPU, double the threads
```

Comment:
Double up the sonar threads because they don't run the CPU at 100% on each thread.  Should speed things up.